crates/library/std/src/context.fe

```diff
@@ -86,7 +87,21 @@ pub struct Context {
         }
     }
 
+    pub fn raw_call(self, addr: address, value: u256, mut buf: MemoryBuffer) {
+        unsafe {
+            assert evm::call(
```

Comment:
Ignore this, if you already thought about this but is `assert` the right thing to do here? I believe we should just return a `bool` here and let the caller handle that. For instance, in the multisig wallet that I'm working on I'll have to inspect the return and need to emit `Success` or `Failure` events based on the outcome. But if `raw_call` does already internally fail with an assertion error, then I believe there would be no way to handle this gracefully, right?

Reply:
returning bool here makes sense